src/main/java/org/folio/circulation/domain/LoanRepository.java

```diff
@@ -303,14 +308,62 @@ private static void keepPatronGroupIdAtCheckoutProperties(Loan loan, JsonObject
     }
 
     final Result<CqlQuery> statusQuery = getStatusCQLQuery("Open");
-    final Result<CqlQuery> itemIdQuery = exactMatchAny("itemId", itemsToFetchLoansFor);
+    final Result<CqlQuery> itemIdQuery = exactMatchAny(ITEM_ID, itemsToFetchLoansFor);
 
     return queryLoanStorage(requests.size(), statusQuery.combine(
         itemIdQuery, CqlQuery::and))
       .thenApply(multipleLoansResult -> multipleLoansResult.next(
         loans -> matchLoansToRequests(multipleRequests, loans)));
   }
 
+  public CompletableFuture<Result<List<InTransitReportEntry>>> fetchLoans(
```

Comment:
This method has a very general name, yet appears to be specific to the in transit report, please move it to a class used specifically for the in transit report.

The loan repository shouldn’t really know about the in transit report or a specific item status.

Reply:
Moved